Index: diaspora/config/boot.rb
===================================================================
--- diaspora.orig/config/boot.rb
+++ diaspora/config/boot.rb
@@ -1,9 +1,163 @@
+#require 'rubygems'
+
 # Set up gems listed in the Gemfile.
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
-require 'bundler/setup' if File.exists?(ENV['BUNDLE_GEMFILE'])
+#ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
+#require 'bundler/setup' if File.exists?(ENV['BUNDLE_GEMFILE'])
 
 # Ensure Builder is loaded
 require 'active_support/builder' unless defined?(Builder)
 
 # Load configuration early
 require_relative 'load_config'
+
+require 'rails'
+require 'active_record'
+require 'action_view'
+
+# API and JSON
+
+require 'acts_as_api'
+require 'json'
+
+# Authentication
+
+require 'devise'
+require 'devise_lastseenable'
+
+# Captcha
+
+require 'simple_captcha2'
+
+# Background processing
+
+require 'sidekiq'
+require 'sinatra'
+
+# Scheduled processing
+
+require 'sidetiq'
+
+# Compression
+
+require 'uglifier'
+
+
+# Configuration
+
+require 'configurate'
+
+# Cross-origin resource sharing
+
+require 'rack/cors'
+
+# CSS
+
+require 'compass-rails'
+require 'bootstrap-sass'
+require 'sass-rails'
+
+
+# Database
+
+require 'pg'
+
+require 'activerecord-import'
+require 'foreigner'
+
+# File uploading
+
+require 'carrierwave'
+require 'fog'
+require 'mini_magick'
+require 'remotipart'
+
+# GUID generation
+
+require 'uuid'
+
+# Icons
+
+require 'entypo-rails'
+
+# JavaScript
+
+require 'backbone-on-rails'
+require 'handlebars_assets'
+require 'jquery-rails'
+require 'rails-assets-jquery' # Should be kept in sync with jquery-rails
+require 'js_image_paths'
+require 'js-routes'
+require 'rails-assets-punycode'
+
+# jQuery plugins
+
+require 'rails-assets-jeresig--jquery.hotkeys'
+require 'rails-assets-jquery-idletimer'
+require 'rails-assets-jquery-placeholder'
+require 'rails-assets-jquery-textchange'
+require 'rails-assets-perfect-scrollbar'
+
+# Localization
+
+require 'http_accept_language'
+require 'i18n-inflector-rails'
+require 'rails-i18n'
+
+# Mail
+
+require 'markerb'
+require 'messagebus_ruby_api'
+
+# Parsing
+
+require 'nokogiri'
+require 'redcarpet'
+#require 'twitter-text'
+require 'roxml'
+require 'oembed'
+require 'opengraph_parser'
+
+# Services
+
+require 'omniauth'
+require 'omniauth-facebook'
+require 'omniauth-tumblr'
+require 'omniauth-twitter'
+require 'twitter'
+require 'omniauth-wordpress'
+
+# Tags
+
+require 'acts-as-taggable-on'
+
+# URIs and HTTP
+
+require 'addressable/uri'
+require 'faraday'
+require 'faraday_middleware'
+require 'typhoeus'
+
+# Views
+
+require 'gon'
+require 'haml'
+require 'mobile-fu'
+require 'will_paginate'
+require 'rails-timeago'
+
+# Workarounds
+# https://github.com/rubyzip/rubyzip#important-note
+require 'zip/zip'
+
+  # Analytics
+
+  require 'rack/google-analytics'
+  require 'rack/piwik'
+
+  # Click-jacking protection
+
+  require 'rack-protection'
+
+  # Redirects
+
+  require 'rack/ssl'
