adapt to jquery-ui-rails 6 in debian

--- a/Gemfile
+++ b/Gemfile
@@ -96,7 +96,7 @@
 gem "backbone-on-rails", "~> 1.2"
 gem "handlebars_assets", "~> 0.23.1"
 gem "jquery-rails",      "~> 4.1", ">= 4.1.1"
-gem "jquery-ui-rails",   "~> 5.0", ">= 5.0.5"
+gem "jquery-ui-rails",   "~> 6.0"
 gem "js_image_paths",    "~> 0.1.0"
 gem "js-routes",         "~> 1.2", ">= 1.2.9"
 
--- a/app/assets/javascripts/main.js
+++ b/app/assets/javascripts/main.js
@@ -15,8 +15,8 @@
 //= require jquery.infinitescroll-custom
 //= require jquery-ui/core
 //= require jquery-ui/widget
-//= require jquery-ui/mouse
-//= require jquery-ui/sortable
+//= require jquery-ui/widgets/mouse
+//= require jquery-ui/widgets/sortable
 //= require keycodes
 //= require fileuploader-custom
 //= require handlebars.runtime
