Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -114,7 +114,7 @@ gem 'nokogiri',         '~> 1.6'
 gem 'redcarpet',        '3.2.0'
 gem 'twitter-text',     '1.10.0'
 gem 'roxml',            '>= 3.1.6'
-gem 'ruby-oembed',      '0.8.11'
+gem 'ruby-oembed',      '~> 0.8'
 gem 'opengraph_parser', '0.2.3'
 
 
