Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -13,7 +13,7 @@ gem "responders", "~> 2.0"
 
 # Appserver
 
-gem "unicorn", "4.9.0", require: false
+gem "unicorn", "~> 4.8", require: false
 
 # API and JSON
 
@@ -79,7 +79,7 @@ gem "uuid", "2.3.7"
 
 # Icons
 
-gem "entypo-rails", "2.2.3"
+gem "entypo-rails", "~> 2.2.2"
 
 # JavaScript
 
@@ -100,7 +100,7 @@ source "https://rails-assets.org" do
   gem "rails-assets-markdown-it--markdown-it-for-inline", "0.1.1"
   gem "rails-assets-markdown-it-sub",                     "1.0.0"
   gem "rails-assets-markdown-it-sup",                     "1.0.0"
-  gem "rails-assets-highlightjs",                         "8.5.0"
+  gem "rails-assets-highlightjs",                         "~> 8.5"
 
   # jQuery plugins
 
@@ -129,7 +129,7 @@ gem "messagebus_ruby_api", ">=1.0.3"
 
 gem "nokogiri",          "1.6.6.2"
 gem "redcarpet",         "3.2.3"
-gem "twitter-text",      "1.12.0"
+gem "twitter-text",      "~> 1.11"
 gem "roxml",             "~> 3.1"
 gem "ruby-oembed",       "0.8.14"
 gem "open_graph_reader", "0.6.1"
@@ -209,7 +209,7 @@ group :production do # we don"t install
 
   # Process management
 
-  gem "eye", "0.7.pre"
+  gem "eye", "~> 0.6"
 
   # Redirects
 
