Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -8,7 +8,7 @@ gem "responders", "~> 2.0"
 
 # Appserver
 
-gem "unicorn", "4.9.0", require: false
+gem "unicorn", "~> 4.8", require: false
 
 # Federation
 
@@ -69,7 +69,7 @@ gem "activerecord-import", "0.8.0"
 # File uploading
 
 gem "carrierwave", "0.10.0"
-gem "fog",         "1.31.0"
+gem "fog",         "~> 1.30"
 gem "mini_magick", "4.2.7"
 gem "remotipart",  "1.2.1"
 
@@ -78,7 +78,7 @@ gem "uuid", "2.3.8"
 
 # Icons
 
-gem "entypo-rails", "2.2.3"
+gem "entypo-rails", "~> 2.2.2"
 
 # JavaScript
 
@@ -92,7 +92,7 @@ gem "js-routes",         "1.0.1"
 source "https://rails-assets.org" do
   gem "rails-assets-jquery",                              "1.11.2" # Should be kept in sync with jquery-rails
 
-  gem "rails-assets-markdown-it",                         "4.4.0"
+  gem "rails-assets-markdown-it",                         "~> 4.2"
   gem "rails-assets-markdown-it-hashtag",                 "0.3.1"
   gem "rails-assets-markdown-it-diaspora-mention",        "0.3.0"
   gem "rails-assets-markdown-it-sanitizer",               "0.3.1"
@@ -207,7 +207,7 @@ group :production do # we don"t install
 
   # Process management
 
-  gem "eye", "0.7.pre"
+  gem "eye", "~> 0.6"
 
   # Redirects
 
