Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -13,7 +13,7 @@ gem "responders", "~> 2.0"
 
 # Appserver
 
-gem "unicorn", "4.9.0", require: false
+gem "unicorn", "~> 4.8", require: false
 
 # API and JSON
 
@@ -79,7 +79,7 @@ gem "uuid", "2.3.7"
 
 # Icons
 
-gem "entypo-rails", "2.2.3"
+gem "entypo-rails", "~> 2.2.2"
 
 # JavaScript
 
@@ -100,7 +100,7 @@ source "https://rails-assets.org" do
   gem "rails-assets-markdown-it--markdown-it-for-inline", "0.1.1"
   gem "rails-assets-markdown-it-sub",                     "1.0.0"
   gem "rails-assets-markdown-it-sup",                     "1.0.0"
-  gem "rails-assets-highlightjs",                         "8.5.0"
+  gem "rails-assets-highlightjs",                         "~> 8.5"
 
   # jQuery plugins
 
