Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -8,7 +8,7 @@ gem "responders", "~> 2.0"
 
 # Appserver
 
-gem "unicorn", "4.9.0", require: false
+gem "unicorn", "~> 4.8", require: false
 
 # Federation
 
@@ -78,7 +78,7 @@ gem "uuid", "2.3.8"
 
 # Icons
 
-gem "entypo-rails", "2.2.3"
+gem "entypo-rails", "~> 2.2.2"
 
 # JavaScript
 
@@ -207,7 +207,7 @@ group :production do # we don"t install
 
   # Process management
 
-  gem "eye", "0.7.pre"
+  gem "eye", "~> 0.6"
 
   # Redirects
 
