Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -13,7 +13,7 @@ gem "responders", "~> 2.0"
 
 # Appserver
 
-gem "unicorn", "4.9.0", require: false
+gem "unicorn", "~> 4.8", require: false
 
 # API and JSON
 
@@ -79,7 +79,7 @@ gem "uuid", "2.3.7"
 
 # Icons
 
-gem "entypo-rails", "2.2.3"
+gem "entypo-rails", "~> 2.2.2"
 
 # JavaScript
 
