Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -132,7 +132,7 @@ gem "open_graph_reader", "0.5.0"
 # Services
 
 gem "omniauth",           "1.2.2"
-gem "omniauth-facebook",  "1.6.0"
+gem "omniauth-facebook",  ">= 1.6.0"
 gem "omniauth-tumblr",    "1.1"
 gem "omniauth-twitter",   "1.0.1"
 gem "twitter",            "4.8.1"
