Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -123,7 +123,7 @@ gem "messagebus_ruby_api", ">=1.0.3"
 # Parsing
 
 gem "nokogiri",          "1.6.6.2"
-gem "redcarpet",         "3.2.2"
+gem "redcarpet",         "~> 3.2.0"
 gem "twitter-text",      "1.11.0"
 gem "roxml",             ">=3.1.6"
 gem "ruby-oembed",       "0.8.12"
@@ -152,7 +152,7 @@ gem "acts-as-taggable-on", "3.5.0"
 
 # URIs and HTTP
 
-gem "addressable",        "2.3.7", require: "addressable/uri"
+gem "addressable",        "~> 2.3.6", require: "addressable/uri"
 gem "faraday",            "0.9.1"
 gem "faraday_middleware", "0.9.1"
 gem "faraday-cookie_jar", "0.0.6"
