Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -56,7 +56,7 @@ gem "rack-cors", "0.4.0", require: "rack
 gem "bootstrap-sass", "2.3.2.2"
 gem "compass-rails",  "2.0.4"
 gem "sass-rails",     "5.0.1"
-gem "autoprefixer-rails", "5.1.11"
+gem "autoprefixer-rails", "~> 5.1.8"
 
 # Database
 
@@ -83,15 +83,15 @@ gem "entypo-rails", "~> 2.2.2"
 
 # JavaScript
 
-gem "backbone-on-rails",                                "1.1.2.1"
+gem "backbone-on-rails",                                "~> 1.1.2"
 gem "handlebars_assets",                                "0.20.1"
 gem "jquery-rails",                                     "4.0.4"
 gem "jquery-ui-rails",                                  "5.0.3"
 gem "js_image_paths",                                   "0.0.2"
-gem "js-routes",                                        "1.0.1"
+gem "js-routes",                                        "~> 1.0.0"
 
 source "https://rails-assets.org" do
-  gem "rails-assets-jquery",                              "1.11.2" # Should be kept in sync with jquery-rails
+  gem "rails-assets-jquery",                              "~> 1.11.1" # Should be kept in sync with jquery-rails
 
   gem "rails-assets-markdown-it",                         "4.2.1"
   gem "rails-assets-markdown-it-hashtag",                 "0.3.1"
@@ -157,8 +157,8 @@ gem "acts-as-taggable-on", "3.5.0"
 
 # URIs and HTTP
 
-gem "addressable",        "2.3.8", require: "addressable/uri"
-gem "faraday",            "0.9.1"
+gem "addressable",        "~> 2.3.6", require: "addressable/uri"
+gem "faraday",            "~> 0.9.0"
 gem "faraday_middleware", "0.9.1"
 gem "faraday-cookie_jar", "0.0.6"
 gem "typhoeus",           "0.7.1"
@@ -166,7 +166,7 @@ gem "typhoeus",           "0.7.1"
 # Views
 
 gem "gon",                     "5.2.3"
-gem "haml",                    "4.0.6"
+gem "haml",                    "~> 4.0.5"
 gem "mobile-fu",               "1.3.1"
 gem "will_paginate",           "3.0.7"
 gem "rails-timeago",           "~>2.11"
@@ -205,7 +205,7 @@ group :production do # we don"t install
 
   # Click-jacking protection
 
-  gem "rack-protection", "1.5.3"
+  gem "rack-protection", "~> 1.5.2"
 
   # Process management
 
@@ -213,7 +213,7 @@ group :production do # we don"t install
 
   # Redirects
 
-  gem "rack-rewrite", "1.5.1", require: false
+  gem "rack-rewrite", "~> 1.5.0", require: false
   gem "rack-ssl",     "1.4.1", require: "rack/ssl"
 
   # Third party asset hosting
