Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -31,7 +31,7 @@ gem "simple_captcha2", "0.3.4", require:
 
 # Background processing
 
-gem "sidekiq", "3.4.1"
+gem "sidekiq", "~> 3.4.1"
 gem "sinatra", "1.4.6"
 
 # Scheduled processing
@@ -55,14 +55,14 @@ gem "rack-cors", "0.4.0", require: "rack
 gem "bootstrap-sass", "2.3.2.2"
 gem "compass-rails",  "2.0.4"
 gem "sass-rails",     "5.0.1"
-gem "autoprefixer-rails", "5.2.1"
+gem "autoprefixer-rails", "~> 5.2.1"
 
 # Database
 
 ENV["DB"] ||= "mysql"
 
 gem "mysql2", "0.3.18" if ENV["DB"] == "all" || ENV["DB"] == "mysql"
-gem "pg",     "0.18.2" if ENV["DB"] == "all" || ENV["DB"] == "postgres"
+gem "pg",     "~> 0.18.1" if ENV["DB"] == "all" || ENV["DB"] == "postgres"
 
 gem "activerecord-import", "0.8.0"
 
@@ -70,11 +70,11 @@ gem "activerecord-import", "0.8.0"
 
 gem "carrierwave", "0.10.0"
 gem "fog",         "~> 1.30"
-gem "mini_magick", "4.2.7"
+gem "mini_magick", "~> 4.2.3"
 gem "remotipart",  "1.2.1"
 
 # GUID generation
-gem "uuid", "2.3.8"
+gem "uuid", "~> 2.3.7"
 
 # Icons
 
@@ -82,15 +82,15 @@ gem "entypo-rails", "~> 2.2.2"
 
 # JavaScript
 
-gem "backbone-on-rails", "1.1.2.1"
-gem "handlebars_assets", "0.20.2"
+gem "backbone-on-rails", "~> 1.1.2"
+gem "handlebars_assets", "~> 0.20.1"
 gem "jquery-rails",      "4.0.4"
-gem "jquery-ui-rails",   "5.0.5"
+gem "jquery-ui-rails",   "~> 5.0.3"
 gem "js_image_paths",    "0.0.2"
-gem "js-routes",         "1.0.1"
+gem "js-routes",         "~> 1.0.0"
 
 source "https://rails-assets.org" do
-  gem "rails-assets-jquery",                              "1.11.2" # Should be kept in sync with jquery-rails
+  gem "rails-assets-jquery",                              "~> 1.11.1" # Should be kept in sync with jquery-rails
 
   gem "rails-assets-markdown-it",                         "~> 4.2"
   gem "rails-assets-markdown-it-hashtag",                 "0.3.1"
@@ -104,7 +104,7 @@ source "https://rails-assets.org" do
   # jQuery plugins
 
   gem "rails-assets-jeresig--jquery.hotkeys",       "0.2.0"
-  gem "rails-assets-jquery-placeholder",            "2.1.2"
+  gem "rails-assets-jquery-placeholder",            "~> 2.1.1"
   gem "rails-assets-jquery-textchange",             "0.2.3"
   gem "rails-assets-perfect-scrollbar",             "0.6.3"
   gem "rails-assets-jakobmattsson--jquery-elastic", "1.6.11"
@@ -155,16 +155,16 @@ gem "acts-as-taggable-on", "3.5.0"
 
 # URIs and HTTP
 
-gem "addressable",        "2.3.8", require: "addressable/uri"
-gem "faraday",            "0.9.1"
+gem "addressable",        "~> 2.3.6", require: "addressable/uri"
+gem "faraday",            "~> 0.9.0"
 gem "faraday_middleware", "0.9.1"
 gem "faraday-cookie_jar", "0.0.6"
-gem "typhoeus",           "0.7.2"
+gem "typhoeus",           "~> 0.7.1"
 
 # Views
 
 gem "gon",                     "5.2.3"
-gem "haml",                    "4.0.6"
+gem "haml",                    "~> 4.0.5"
 gem "mobile-fu",               "1.3.1"
 gem "will_paginate",           "3.0.7"
 gem "rails-timeago",           "~>2.11"
@@ -175,7 +175,7 @@ gem "logging-rails", "0.5.0", require: "
 
 # Reading and writing zip files
 
-gem "rubyzip", "1.1.7"
+gem "rubyzip", "~> 1.1.6"
 
 # Prevent occasions where minitest is not bundled in
 # packaged versions of ruby. See following issues/prs:
@@ -194,7 +194,7 @@ gem "minitest"
 group :production do # we don"t install these on travis to speed up test runs
   # Administration
 
-  gem "rails_admin", "0.6.8"
+  gem "rails_admin", "~> 0.6.7"
 
   # Analytics
 
@@ -203,7 +203,7 @@ group :production do # we don"t install
 
   # Click-jacking protection
 
-  gem "rack-protection", "1.5.3"
+  gem "rack-protection", "~> 1.5.2"
 
   # Process management
 
@@ -211,7 +211,7 @@ group :production do # we don"t install
 
   # Redirects
 
-  gem "rack-rewrite", "1.5.1", require: false
+  gem "rack-rewrite", "~> 1.5.0", require: false
   gem "rack-ssl",     "1.4.1", require: "rack/ssl"
 
   # Third party asset hosting
