Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -54,9 +54,9 @@ gem "rack-cors", "0.4.0", require: "rack
 # CSS
 
 gem "bootstrap-sass", "2.3.2.2"
-gem "compass-rails",  "2.0.4"
-gem "sass-rails",     "5.0.1"
-gem "autoprefixer-rails", "5.2.1.1"
+gem "compass-rails",  "~> 2.0.4"
+gem "sass-rails",     "~> 5.0.1"
+gem "autoprefixer-rails", "~> 5.2.1"
 
 # Database
 
@@ -91,7 +91,7 @@ gem "js_image_paths",    "0.0.2"
 gem "js-routes",         "1.0.1"
 
 source "https://rails-assets.org" do
-  gem "rails-assets-jquery",                              "1.11.2" # Should be kept in sync with jquery-rails
+  gem "rails-assets-jquery",                              "~> 1.11.1" # Should be kept in sync with jquery-rails
 
   gem "rails-assets-markdown-it",                         "4.4.0"
   gem "rails-assets-markdown-it-hashtag",                 "0.3.1"
@@ -168,7 +168,7 @@ gem "gon",                     "6.0.1"
 gem "haml",                    "4.0.6"
 gem "mobile-fu",               "1.3.1"
 gem "will_paginate",           "3.0.7"
-gem "rails-timeago",           "2.11.0"
+gem "rails-timeago",           "~> 2.11.0"
 
 # Logging
 
