Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -52,7 +52,7 @@ gem "rack-cors", "0.4.0", require: "rack
 
 # CSS
 
-gem "bootstrap-sass", "2.3.2.2"
+gem "bootstrap-sass", "2.3.2.2", path: 'vendor/gems/bootstrap-sass-2.3.2.2'
 gem "compass-rails",  "2.0.4"
 gem "sass-rails",     "~> 5.0.1"
 gem "autoprefixer-rails", "~> 5.2.1"
