Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -53,7 +53,7 @@ gem "rack-cors", "0.4.0", require: "rack
 
 # CSS
 
-gem "bootstrap-sass", "2.3.2.2"
+gem "bootstrap-sass", "2.3.2.2", path: 'vendor/gems/bootstrap-sass-2.3.2.2'
 gem "compass-rails",  "2.0.5"
 gem "sass-rails",     "5.0.3"
 gem "autoprefixer-rails", "6.0.3"
