to fix undefined method `raise_in_transactional_callbacks=' for ActiveRecord::Base:Class

Index: diaspora/config/application.rb
===================================================================
--- diaspora.orig/config/application.rb
+++ diaspora/config/application.rb
@@ -101,7 +101,7 @@ module Diaspora
     end
 
     # Will be default with Rails 5
-    config.active_record.raise_in_transactional_callbacks = true
+    #config.active_record.raise_in_transactional_callbacks = true
 
     # Setup action mailer early
     config.action_mailer.default_url_options = {
