Index: diaspora/config/application.rb
===================================================================
--- diaspora.orig/config/application.rb
+++ diaspora/config/application.rb
@@ -2,6 +2,7 @@ require_relative 'boot'
 
 require 'rails/all'
 Bundler.require(:default, Rails.env)
+require 'foreigner'
 
 # Do not dump the limit of boolean fields on MySQL,
 # since that generates a db/schema.rb that's incompatible
