Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=809707

Index: diaspora/app/assets/javascripts/jsxc.js
===================================================================
--- diaspora.orig/app/assets/javascripts/jsxc.js
+++ diaspora/app/assets/javascripts/jsxc.js
@@ -1,6 +1,6 @@
 //= require jquery.slimscroll/jquery.slimscroll
 //= require jquery-colorbox
-//= require favico.js
+//= require favico.js/favico.js
 //= require jquery-fullscreen-plugin
 //= require diaspora_jsxc
 
