relaxing patch releases required for newer diaspora versions

Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -22,7 +22,7 @@ gem "json-schema", "~> 2.5"
 
 # Authentication
 
-gem "devise", "3.5.2"
+gem "devise", "~> 3.5.2"
 gem "devise_lastseenable", "0.0.6"
 gem "devise-token_authenticatable", "~> 0.4.0"
 
@@ -33,7 +33,7 @@ gem "simple_captcha2", "0.3.4", require:
 # Background processing
 
 gem "sidekiq", "4.0.1"
-gem "sinatra", "1.4.6"
+gem "sinatra", "~> 1.4.6"
 
 # Scheduled processing
 
@@ -63,7 +63,7 @@ gem "autoprefixer-rails", "~> 6.0"
 ENV["DB"] ||= "mysql"
 
 gem "mysql2", "0.3.20" if ENV["DB"] == "all" || ENV["DB"] == "mysql"
-gem "pg",     "0.18.3" if ENV["DB"] == "all" || ENV["DB"] == "postgres"
+gem "pg",     "~> 0.18.3" if ENV["DB"] == "all" || ENV["DB"] == "postgres"
 
 gem "activerecord-import", "0.10.0"
 
