relaxing patch releases required for newer diaspora versions

Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -22,7 +22,7 @@ gem "json-schema", "~> 2.6",
 
 # Authentication
 
-gem "devise", "3.5.6"
+gem "devise", "~> 3.5.6"
 gem "devise_lastseenable", "0.0.6"
 gem "devise-token_authenticatable", "~> 0.4.0"
 
@@ -33,7 +33,7 @@ gem "simple_captcha2", "0.4.0", require:
 # Background processing
 
 gem "sidekiq", "~> 4.0"
-gem "sinatra", "1.4.7"
+gem "sinatra", "~> 1.4.7"
 
 # Scheduled processing
 
