Index: diaspora/Gemfile
===================================================================
--- diaspora.orig/Gemfile
+++ diaspora/Gemfile
@@ -85,7 +85,7 @@ gem "entypo-rails", "~> 2.2", ">= 2.2.3"
 
 gem "backbone-on-rails", "~> 1.2"
 gem "handlebars_assets", "~> 0.23.0"
-gem "jquery-rails",      "~> 4.1"
+gem "jquery-rails",      "~> 4.0"
 gem "jquery-ui-rails",   "~> 5.0", ">= 5.0.5"
 gem "js_image_paths",    "~> 0.1.0"
 gem "js-routes",         "~> 1.2", ">= 1.2.4"
