jquery is backword compatible

--- a/Gemfile
+++ b/Gemfile
@@ -90,13 +90,13 @@
 
 gem "backbone-on-rails", "~> 1.2"
 gem "handlebars_assets", "~> 0.23.1"
-gem "jquery-rails",      "~> 4.1", ">= 4.1.1"
+gem "jquery-rails",      "~> 4.0"
 gem "jquery-ui-rails",   "~> 5.0", ">= 5.0.5"
 gem "js_image_paths",    "~> 0.1.0"
 gem "js-routes",         "~> 1.2", ">= 1.2.9"
 
 source "https://rails-assets.org" do
-  gem "rails-assets-jquery",                              "~> 2.2", ">= 2.2.1" # Should be kept in sync with jquery-rails
+  gem "rails-assets-jquery",                              ">= 1.11.2" # Should be kept in sync with jquery-rails
 
   gem "rails-assets-markdown-it",                         "~> 7.0"
   gem "rails-assets-markdown-it-hashtag",                 "~> 0.4.0"
