Description: Use newer faraday versions
Bug: https://github.com/diaspora/diaspora/issues/4950

--- a/Gemfile
+++ b/Gemfile
@@ -90,7 +90,7 @@
 # URIs and HTTP
 
 gem 'addressable',        '2.3.5', :require => 'addressable/uri'
-gem 'faraday',            '0.8.9'
+gem 'faraday',            '>= 0.8.9'
 gem 'faraday_middleware', '0.9.0'
 gem 'typhoeus',           '0.6.7'
 
--- a/config/initializers/faraday.rb
+++ b/config/initializers/faraday.rb
@@ -2,7 +2,6 @@
 #   licensed under the Affero General Public License version 3 or later.  See
 #   the COPYRIGHT file.
 options = {
-  timeout: 25,
   ssl: {
     ca_file: AppConfig.environment.certificate_authorities.get
   }
